plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -132,6 +129,10 @@ function buildDefaultIdentityClient({
           throw new TypeError('Expected string sub claim');
         }
 
+        if (sub === 'backstage-server') {
+          return undefined;
```

Comment:
Noting that eventually we'll want to validate and forward the server identity to the catalog etc. here instead of throwing away the identity. For now this is better than throwing